Release 1.6 (#76)
spinupwp/spinupwp-plugin: readme.txt

== Changelog ==

= 1.6 (2024-02-06) =

> * Add option to purge current URL.

Reviewer: ```suggestion
* New: Add "Purge this URL" option to our Cache menu in the WordPress nav bar.
```

> * Increase default timeout limits from 1 to 5 seconds.

Reviewer: ```suggestion
* Improvement: Increase default cache purge timeout limit from 1 to 5 seconds.
```

> * Allow cache key customization.

Reviewer: ```suggestion
* New: Cache key customization. /props quimcastella
```

> * Fix 'Clear All Cache' page cache functionality when Redis errors.

Reviewer: ```suggestion
* Bug Fix: Page cache not cleared when clearing object cache fails.
```

> * Fix "Deprecated: file_exists()" on PHP 8.1.

Reviewer: ```suggestion
* Improvement: PHP 8.1 compatibility. /props afragen
```
